http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java

```diff
@@ -117,12 +117,20 @@
     public static final SdkHttpConfigurationOption<TlsTrustManagersProvider> TLS_TRUST_MANAGERS_PROVIDER =
         new SdkHttpConfigurationOption<>("TlsTrustManagersProvider", TlsTrustManagersProvider.class);
 
+    /**
+     * An upper limit on how long from the time a TLS handshake is allowed to take from the time the CLIENT HELLO message is
+     * sent to the time the client and server have fully negotiated ciphers and exchanged keys.
```

Comment:
I can't understand exactly what the text means, but maybe it's the `from the time` on line 121 that should just be removed?

Reply:
Yeah, copy paste error, will fix